remark/formatTableCells.test.ts

```diff
@@ -1,3 +1,4 @@
+// @ts-expect-error
```

Comment:
Previously, you got an error in Code (depends on your editor) however linting passed. This just ignores the error for warning/error for lint and editor